api/src/main/java/org/openmrs/api/ConditionVerificationStatus.java

```diff
@@ -0,0 +1,20 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.api;
+
+/**
+ * The ConditionVerificationStatus is what we've historically called "diagnosis certainty" and what FHIR refers to as
+ * ConditionVerificationStatus. This is the verification status to support or decline the clinical status of the condition or
+ * diagnosis.
+ */
+public enum ConditionVerificationStatus {
+	PROVISIONAL, // This is a tentative diagnosis. This Was called "Presumed" in the original EMRAPI module implementation
```

Comment:
Can you do some googling on how to do javadoc for enum values?

Reply:
Alright. I am on it.